src/Query/Builder.php

```diff
@@ -209,12 +212,14 @@ public function whereKey($id)
     /**
      * Add an $expand clause to the query.
      *
-     * @param  string  $table
-     * @param  string  $first
-     * @param  string  $operator
-     * @param  string  $second
-     * @param  string  $type
-     * @param  bool    $where
+     * @param string $property
+     * @param string $first
+     * @param string $operator
+     * @param string $second
+     * @param string $type
+     * @param bool   $ref
+     * @param bool   $count
+     *
      * @return $this
      */
     public function expand($property, $first, $operator = null, $second = null, $type = 'inner', $ref = false, $count = false)
```

Comment:
good catch. Yes, `$property` is more correct terminology for OData vs `$table` and I forgot to update the docblock. Let's go with `$property`.